src/ORM/DataObject.php

```diff
@@ -3175,14 +3175,24 @@ public static function reset()
     /**
      * Return the given element, searching by ID
      *
-     * @param string $callerClass The class of the object to be returned
+     * @param string $callerClass The class of the object to be returned (optional if called from extending class)
```

Comment:
Rename to `$callerClassOrID` to reflect the duality of the argument.

Reply:
We can't do this without breaking semver - unless you're just talking about the docblock?

Reply:
Renaming params are not semver violations

Reply:
True, you're right